bindinfo/handle.go

```diff
@@ -356,15 +364,18 @@ func (c cache) removeDeletedBindMeta(hash string, meta *BindMeta) {
 }
 
 // removeStaleBindMetas removes all the stale BindMeta in the cache.
-func (c cache) removeStaleBindMetas(hash string, meta *BindMeta) {
+func (c cache) removeStaleBindMetas(hash string, meta *BindMeta, scope string) {
 	metas, ok := c[hash]
 	if !ok {
 		return
 	}
 
-	// remove stale bindMetas.
 	for i := len(metas) - 1; i >= 0; i-- {
 		if metas[i].isStale(meta) {
+			metrics.BindMemoryUsage.WithLabelValues(scope).Sub(metas[i].size())
+			if metas[i].Status == Using {
```

Comment:
should we decrease the counter no matter whether it is `Using`?

Reply:
No, currently it only means valid binds, and for global handles, it could only be `using`.